src/actions/restActions.js

```diff
@@ -0,0 +1,23 @@
+import responseHandler from './responseHandler'
+
+
+const base_url = 'http://localhost:9292/'
+const reset_game_url = base_url+'reset_game';
+const play_url = base_url+'play';
```

Comment:
Just a reminder from our session, this could be refactored into using environment variables so that you won't have to modify the test when the environment changes.